app/serializers/move_serializer.rb

```diff
@@ -22,16 +22,17 @@ class MoveSerializer
              :date_from,
              :date_to
 
-  has_one :person
-  has_one :profile
+  belongs_to :person
```

Comment:
I think this is a `has_one through` relationship on the move model

Reply:
Good catch - I'll revert this one.